src/ResourceManager/Resources/Commands.Resources/help/New-AzADUser.md

```diff
@@ -14,8 +14,7 @@ Creates a new active directory user.
 ## SYNTAX
 
 ```
-New-AzADUser -DisplayName <String> -UserPrincipalName <String> -Password <SecureString> [-ImmutableId <String>]
- [-MailNickname <String>] [-ForceChangePasswordNextLogin] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+New-AzADUser -DisplayName <String> -UserPrincipalName <String> -Password <SecureString> -MailNickname <String> [-ImmutableId <String>] [-ForceChangePasswordNextLogin] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
```

Comment:
This parameter is not required in the cmdlet code - are you sayign that it is actually required in some circumstances, or is this an oversight?

Reply:
I get an error if I do not provide it:
```
PS C:\svn\obs\azure\scripts> $SecureStringPassword = ConvertTo-SecureString -String "TxQl77FS*#m9" -AsPlainText -Force
New-AzADUser -DisplayName "dnhf01-test01" -UserPrincipalName "dnhf01-test03@xyz.onmicrosoft.com" -Password $SecureStringPassword
New-AzADUser : 'MailNickname' cannot be null.
At line:2 char:1
+ New-AzADUser -DisplayName "dnhf01-test01" -UserPrincipalName "dnhf01- ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [New-AzADUser], ValidationException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.ActiveDirectory.NewAzureADUserCommand
```

Reply:
I use version 1.0.1:
```
PS C:\svn\obs\azure\scripts> Get-Module Az

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     1.0.1      Az
```

Reply:
So in that case, there is a coding issue, which will require more than a help fix - this cmdlet should be marked as required if it is actually required.